pyproject.toml

```diff
@@ -52,6 +52,7 @@ pandas = "^2.1.0"
 pgvector = "^0.2.3"
 psycopg2-binary = "^2.9.9"
 snowflake-connector-python = "^3.2"
+pytimeparse = "^1.1.8"
```

Comment:
Let's instead use [dateparser](https://dateparser.readthedocs.io/en/latest/) instead as we're already using it one of the tools.